tests/testthat/_snaps/create_codemeta/cm2.json

```diff
@@ -160,9 +160,9 @@
   {
     "@context": ["https://doi.org/10.5063/schema/codemeta-2.0", "http://schema.org"],
     "@type": "SoftwareSourceCode",
-    "identifier": "pkg1",
+    "identifier": "pkg2",
```

Comment:
Most of these changes stem from the fact that packages are now named liked the folder...